datacenterlight/templates/datacenterlight/faq.html

```diff
@@ -0,0 +1,29 @@
+{% extends "datacenterlight/base.html" %}
+{% load staticfiles cms_tags %}
+{% block content %}
+<div class="full-whydcl-sec">
+    <div class="whydcl-header whydcl-section" id="why_dcl">
```

Comment:
I also suggest to rename the class or add a new name in the css to make it more generic